### PR TITLE
[CDAP-13620] Allow stopping program runs in 'PENDING' and 'STARTING' states

### PR DESCRIPTION
**Pull request by tonybach**
JIRA: https://issues.cask.co/browse/CDAP-13620

- Allow stopping program runs in `PENDING` and `STARTING` states. Before, we disabled the stop button until the run is in `RUNNING` state
- Fixes 'invalid date' showing up as the `Start time` in the stop runs popover, since for `PENDING` and `STARTING` runs there's no `run.start`
- Fixes 'Stop' button not aligning correctly when there are multiple active runs
- Fixes run error not closing even after clicking on the 'X'

### CODE REVIEW

File: cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/PipelineStopButton/index.js [hunk 1]
```diff
@@ -40,15 +40,19 @@ export default class PipelineStopButton extends Component {
 
   state = {
     disabled: true,
-    runningRuns: []
+    activeRuns: []
   };
 
+  active_statuses = ['PENDING', 'STARTING', 'RUNNING'];
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/PipelineStopButton/index.js]: nit: use camelCase unless you are defining a constant with all caps

**Review by elfenheart — APPROVED** [review]
👍 code LGTM

**Comment by leacd** [comment]
LGTM

**Comment by tonybach** [comment]
Green build https://builds.cask.co/browse/CDAP-URUT35-3, merging.

**Pull request merged**